src/app/views/discover/games/list/list.ts

```diff
@@ -106,98 +132,148 @@ export default class RouteDiscoverGamesList extends BaseRouteComponent {
+			case 'scifi':
+				return this.$gettext('science fiction games');
+			case 'retro':
+				return this.$gettext('nostalgic and retro games');
+			case 'pointnclick':
+				return this.$gettext('point and click games');
+			case 'altgame':
+				return this.$gettext('alt games');
+			case 'roguelike':
+				return this.$gettext('rogue like games');
 			default:
-				return this.$gettext('Browsing games');
+				return this.$gettextInterpolate(`%{ category } games`, {
+					category: this.tag,
+				});
 		}
 	}
 
 	/**
 	 * This function doesn't translate, but it's only used for meta tags
 	 * currently.
 	 */
-	get routeMeta() {
-		let descriptiveTag = `games`;
-		if (this.tag === 'fangame') {
-			descriptiveTag = `fangames`;
-		} else if (this.tag === 'scifi') {
-			descriptiveTag = `sci-fi games`;
-		} else if (this.tag === 'rpg') {
-			descriptiveTag = `rpgs`;
-		} else if (this.tag === 'rpg') {
-			descriptiveTag = `altgames`;
-		} else if (this.tag === 'fnaf') {
-			descriptiveTag = `five nights at freddy's fangames`;
-		} else if (this.tag) {
-			descriptiveTag = `${this.tag} games`;
-		}
-
+	get listDescription() {
 		if (!this.section) {
-			return `Browse the freshest ${descriptiveTag} on Game Jolt. They're almost too hot!`;
+			return this.$gettextInterpolate(`Browse the trending %{ gamesType } on Game Jolt.`, {
+				gamesType: this.displayGamesType,
+			});
 		} else if (this.section === 'new') {
-			return `Browse new ${descriptiveTag} on Game Jolt. Good or bad, you decide!`;
+			return this.$gettextInterpolate(
+				// tslint:disable-next-line:max-line-length
+				`Browse new %{ gamesType } on Game Jolt. You can find anything here from hidden gems to doodoo land mines!`,
+				{ gamesType: this.displayGamesType }
+			);
 		} else if (this.section === 'featured') {
-			return `Browse unique and alternative ${descriptiveTag} that we have featured on Game Jolt.`;
+			return this.$gettextInterpolate(
+				`Browse our featured list of %{ gamesType }. A unique and alternative gaming experience awaits.`,
+				{ gamesType: this.displayGamesType }
+			);
 		} else if (this.section === 'best') {
-			return `Browse the best ${descriptiveTag} on Game Jolt.`;
+			return this.$gettextInterpolate(
+				`Browse the best %{ gamesType }, top rated by our community on Game Jolt.`,
+				{
+					gamesType: this.displayGamesType,
+				}
+			);
 		} else if (this.section === 'worst') {
-			return `The worst ${descriptiveTag} decided by the Game Jolt community.`;
+			return this.$gettextInterpolate(
+				`The worst %{ gamesType } decided by the Game Jolt community.`,
+				{ gamesType: this.displayGamesType }
+			);
 		}
 
-		return null;
+		return '';
+	}
+
+	get routeMeta() {
+		if (!this.listing) {
+			return this.listDescription;
+		}
+
+		if (!Array.isArray(this.listing.games) || this.listing.games.length === 0) {
+			return this.listDescription;
+		}
+
+		const count = fuzzynumber(this.listing.gamesCount);
+		const gameTitles = this.listing.games
```

Comment:
Let's randomize this. shuffle them, then pull top 5

Reply:
im not sure if we get devalued for the exact same page with differing description. i'll try finding info on this

Reply:
I just think it'd be nice so that Google always gets a new list of games that are on the page. It's what we did at FC, so I think it's probably smart to do.